index.html

```diff
@@ -148,7 +148,7 @@
 			<div class="sub-description-sub-heading">
 				Ideation
 			</div>
-			<hr class="yellow-accent-line">
+			<hr class="yellow-accent-line-left-sub">
 			<p>We are building a <span id="home-description-emphasis-words">biological band-pass filter</span>. While biological band pass filters have been developed before, none respond to frequency-based inputs but instead respond to amplitude based inputs. By controlling the rates of degradation of the components of our system, we are putting together a <span id="home-description-emphasis-words">frequency-response</span> low-pass and high-pass filter to create a <span id="home-description-emphasis-words">novel</span> band-pass filter.</p>
 			</div>
```

Comment:
Descriptions on the right should be right-aligned (refer to the home page design in "Final Designs" on drive).